tests/helper_utils/__init__.py

```diff
@@ -4,14 +4,16 @@
 from typing import Union
 
 
-def assert_directories_equal(dir_a, dir_b):
+def assert_directories_equal(dir_a, dir_b, ignore_files=[]):
```

Comment:
I just remembered that setting an array as default for a function is a bad practice (my bad, I was the one that wrote this). Although we're not mutating the default array in this function, do we want to still fix that?

Reply:
I noticed it, but kept it in solely for the reason you mention - we're not mutating the array. I'm in full agreement we should fix this, it is by no means critical given the current code base so I'll propose a minor fix in a follow-up.